tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -549,6 +549,7 @@
                   "codec_type": "AUTO",
                   "stat_prefix": "stats",
                   "route_config": {
+                    "name": "static-prometheus:15090",
```

Comment:
in the RDS response this is normally just a name like `15090` right ? Should we just make this 15090? What does the output of proxy-config routes look like withthis change?

Reply:
after change:
```
$ istioctl pc route istio-ingressgateway-545c79dc67-x8h8j.istio-system  
NOTE: This output only contains routes loaded via RDS.
NAME     VIRTUAL HOSTS
static.prometheus:15090         1
```
It might better to add a prefix for it, cause we get this item from envoy static config, it's different from dynamic config.